elixir/apps/web/lib/web/live/actors/index.ex

```diff
@@ -48,14 +48,20 @@ defmodule Web.Actors.Index do
     <.section>
       <:title><%= @page_title %></:title>
 
+      <:action>
+        <.docs_action path="/deploy/actors" />
```

Comment:
I think this path needs to be `/deploy/users`.

Reply:
Thanks!